jsk_pcl_ros_utils/src/colorize_distance_from_plane_nodelet.cpp

```diff
@@ -57,6 +57,10 @@ namespace jsk_pcl_ros_utils
     srv_->setCallback (f);
   }
 
+  ColorizeDistanceFromPlane::~ColorizeDistanceFromPlane() {
```

Comment:
If  we basically add `sync_.reset()` to all, how about define someting like
```
  class Recognition?Filter?Nodelet???: public jsk_topic_tools::DiagnosticNodelet {
       boost::shared_ptr<message_filters::Synchronizer<SyncPolicy> > sync_;
       Recognition?Filter?Nodelet???() {
               sync_ = boost::make_shared<message_filters::Synchronizer<SyncPolicy> >(100);
      }
      ~Recognition?Filter?Nodelet???() {
           sync_.reset(); 
      }
  };

  class PolygonPointsSampler: public jsk_pcl_ros_utils::Recognition?Filter?Nodelet???
```
?? may be you already considered this, and find some drawbacks.

Reply:
I have considered it. 
However, some programs can have multiple asyncs and syncs.
https://github.com/jsk-ros-pkg/jsk_recognition/blob/659a25c7bed3c8786aefa6ef1ce16a9bd53aaecb/jsk_pcl_ros/include/jsk_pcl_ros/ppf_registration.h#L91-L93
We believe that defining a class like this reduces flexibility.
So I added a comment to all programs that use synchronizer.